.coderabbit.yaml

```
@@ -0,0 +1,41 @@
+    - "!**/package-lock.json"
+    - "!**/yarn.lock"
+    - "!**/node_modules/**"
+    - "!**/.env*"
+      
+# Language-specific settings
+language_settings:
+  javascript_and_typescript:
+    enable_eslint: true
+  python:
+    enable_pylint: true
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)